internal/app/app.go

```diff
@@ -44,7 +44,7 @@ type DynamoClient interface {
 	OneByPartialSk(ctx context.Context, pk, partialSk string, v interface{}) error
 	AllByPartialSk(ctx context.Context, pk, partialSk string, v interface{}) error
 	LatestForActor(ctx context.Context, sk string, v interface{}) error
-	AllForActor(ctx context.Context, sk string, v interface{}) error
+	AllBySK(ctx context.Context, sk string, v interface{}) error
```

Comment:
do you prefer the naming `AllBySK` or `AllBySk`? would be good to be consistent either way, and if it is changing the others to uppercase leave it for a separate PR

Reply:
We spoke a while back about capitalising so let's go with that. I'll pick up next.